modules/standard/Math.chpl

```diff
@@ -473,16 +473,14 @@ module Math {
     return chpl_divfloorpos(x, y);
   }
 
-  /* Returns the error function of the argument `x`. This is equivalent to the
-     integral of :proc:`twiceReciprSqrtPi`\ * :proc:`exp`\(`-t**2`)dt from 0
-     to `x`. */
+  /* Returns the error function of the argument `x`. This is equivalent to
+     2/sqrt(pi) * the integral of exp(-t**2)dt from 0 to `x`. */
```

Comment:
Since we're not going to have them be links, let's surround `2/sqrt(pi)` and `exp(-t**2)dt` in double ` so they're more visible in the generated documentation

Reply:
Done

Reply:
It was a puzzle for me to know what style to use (nothing, single `, double `, links). I don't think there's a clearly defined style guide for the whole chapel doc, rigth?

Reply:
Thank you for the links! This will help me the next time I update the doc.